docs/install-machine.md

```diff
@@ -57,6 +57,27 @@ to your PATH.
         $ docker-machine -v
         machine version 0.5.0 (3e06852)
 
+## Installing bash completion scripts
+
+The Machine repository supplies several `bash` scripts that add features such
```

Comment:
Ah, I see it's the opposite of what @moxiegirl recommended.  No need to change it then.